templates/web/base/around/postcode_form.html

```diff
@@ -33,7 +33,10 @@
             IF c.get_param('filter_category'); link_params.filter_category = c.get_param('filter_category'); END;
             IF c.get_param('filter_group'); link_params.filter_group = c.get_param('filter_group'); END;
         %]
-        <a href="[% c.uri_for('/around', link_params) | html %]" id="geolocate_link"> [% loc('Use my current location') %]</a>
+        
+        <a href="[% c.uri_for('/around', link_params) | html %]" id="geolocate_link"><svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19.83 4.17c.17.17.218.429.12.65l-6.594 14.834a.583.583 0 01-1.069-.008L9.91 14.091l-5.557-2.38a.583.583 0 01-.006-1.068L19.18 4.05a.583.583 0 01.648.12h.001z" fill="#000" fill-rule="nonzero"/></svg> 
```

Comment:
Just had a thought! If we did the SVG as an `[% INCLUDE %]`, then it’d be much easier for cobrands to override with their own icon, without needing to fork the entire `postcode_form.html` template.